client/my-sites/upgrades/cart/cart-body.jsx

```diff
@@ -1,50 +1,56 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-var CartItems = require( './cart-items' ),
-	CartCoupon = require( './cart-coupon' ),
-	CartTotal = require( './cart-total' );
-
-var CartBody = React.createClass( {
-	propTypes: {
-		collapse: React.PropTypes.bool
-	},
-	getDefaultProps: function() {
-		return {
-			collapse: false,
-			showCoupon: false
-		};
-	},
-
-	render: function() {
-		if ( ! this.props.cart.hasLoadedFromServer ) {
-			return <div className="cart-body">{ this.translate( 'Loading…', { context: 'Upgrades: Loading cart' } ) }</div>;
+import CartItems from './cart-items';
+import CartCoupon from './cart-coupon';
+import CartTotal from './cart-total';
+
+class CartBody extends Component {
+	static propTypes = {
+		collapse: PropTypes.bool
+	};
+
+	static defaultProps = {
+		collapse: false,
+		showCoupon: false
+	};
+
+	optionalCoupon() {
+		if ( ! this.props.showCoupon ) {
+			return;
+		}
+
+		return <CartCoupon cart={ this.props.cart } />;
+	}
+
+	render() {
+		const { translate, cart } = this.props;
+
+		if ( ! cart.hasLoadedFromServer ) {
+			return <div className="cart-body">
+				{ translate( 'Loading…', { context: 'Upgrades: Loading cart' } ) }
```

Comment:
Sure, there's an example in the [i18n-calyspo docs](https://github.com/Automattic/api-docs-internal/pull/23#issuecomment-244084026):

```
// add a comment to the translator
var content = i18n.translate( 'g:i:s a', {
        comment: 'draft saved date format, see http://php.net/date'
    } );
```

Just use `comment:`where you would use `context:`, and you should be good to go.  Comments are closely analogous to code comments, and just as yummy.

Let us know before you do any mass converting, though, because it flows through the translation system.

If there's already a matching translation without a context (like in this case), it's no problem at all.  The entry with the context will get absorbed into the context-less entry, it will already be translated for users and translators don't get involved.

However, if it's a unique translation, removing the context will delete the translation until the string gets re-translated, so users will see untranslated content and translators will do extra work.